rockcraft/oci.py

```diff
@@ -36,6 +36,12 @@
 
 logger = logging.getLogger(__name__)
 
+# Public Amazon Elastic Container Registry has images that are updated more frequently
+# and with less severe pull-rate limits than Docker Hub.
+ECR_URL = "public.ecr.aws/ubuntu"
```

Comment:
Why did you add this variable and not just do REGISTRY_URL = "public.ecr.aws/ubuntu" I don't see the value of this variable.